packages/dicomImageLoader/src/imageLoader/wadors/metaData/getSequenceItems.ts

```diff
@@ -0,0 +1,19 @@
+import { WADORSMetaDataElement } from 'dicomImageLoader/src/types';
+
+function getSequenceItems(element: any): WADORSMetaDataElement[] {
+  if (!element) {
+    return;
+  }
+  // Value is not present if the attribute has a zero length value
+  if (!element.Value) {
```

Comment:
Use !element.Value?.length since some DICOMweb systems are not quite compliant, and that is a more general test.